plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/jdbc/IcebergJdbcClient.java

```diff
@@ -47,6 +60,22 @@ public void createTable(String schemaName, String tableName, String metadataLoca
         }
     }
 
+    private void createTableV1(String schemaName, String tableName, String metadataLocation)
+    {
+        try (Handle handle = Jdbi.open(connectionFactory)) {
+            handle.createUpdate("" +
+                            "INSERT INTO iceberg_tables " +
+                            "(catalog_name, table_namespace, table_name, metadata_location, previous_metadata_location, iceberg_type) " +
+                            "VALUES (:catalog, :schema, :table, :metadata_location, null, :iceberg_type)")
+                    .bind("catalog", catalogName)
+                    .bind("schema", schemaName)
+                    .bind("table", tableName)
+                    .bind("iceberg_type", "TABLE")
```

Comment:
Can we inline this constant in INSERT statement?

Reply:
Done!